Makefile

```diff
@@ -34,9 +34,15 @@ CHECKSUMS += $(BUNDLE).sha256
 
 VERSION_PACKAGE := $(REPOPATH)/pkg/rakkess/version
 
+DATE_FMT = %Y-%m-%dT%H:%M:%SZ
+ifdef SOURCE_DATE_EPOCH
+    BUILD_DATE ?= $(shell date -u -d "@$(SOURCE_DATE_EPOCH)" "+$(DATE_FMT)" 2>/dev/null || date -u -r "$(SOURCE_DATE_EPOCH)" "+$(DATE_FMT)" 2>/dev/null || date -u "+$(DATE_FMT)")
```

Comment:
I'm a bit worried about all the alternatives here. Quoting the spec for `SOURCE_DATE_EPOCH`:

> The value MUST be an ASCII representation of an integer with no fractional component, identical to the output format of date +%s.

That seems to be covered by the first alternative.

The second looks as if it is using a reference file to obtain a timestamp. As this is not specified by the spec, why should we do that?

Finally, the last alternative seems to be a fallback in case the other ones failed. Here I would rather fail fast and fail the build instead of pretending that everything went fine. WDYT?

Reply:
This is to handle different variants of `date` - the first is for GNU date as used in every GNU/Linux distribution, the 2nd for [FreeBSD date](https://www.unix.com/man-page/FreeBSD/1/date/) and the 3rd one as fallback (e.g. DOS and Solaris date) that have no way to make the output reproducible.

Reply:
If you think, that this program will never be built outside of GNU/Linux, we can simplify the code.

Reply:
I see. So I think the last variant is obsolete, because on these systems people should simply not set the `SOURCE_DATE_EPOCH` env var and the build will do as expected.
And for the two remaining alternatives, a comment would be nice, such as

> GNU and BSD date require different options for a fixed date

I don't mind if the comment is in code or in the commit message, whatever you prefer.